scripts/network/workloads/network-client.toml

```
@@ -5,8 +5,10 @@ per_core = false
 [workload]
 type = "network"
 server = false
-address = [223, 42, 0, 1]
+address = "223.42.0.1"
 target_port = 1337
 arrival_rate = 0.1
 departure_rate = 0.1
-nconnections = 100
+n_static_connections = 100
+n_dyn_connections = 1000
```

Proposing a slight change to naming here based on something I read a ways back ([tiger style programming](https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/TIGER_STYLE.md))

```
Add units or qualifiers to variable names, and put the units or qualifiers last, sorted by descending significance, so that the variable starts with the most significant word, and ends with the least significant word. For example, latency_ms_max rather than max_latency_ms. This will then line up nicely when latency_ms_min is added, as well as group all variables that relate to latency.
```

```suggestion
connections_static_max = 100
connections_dyn_max = 1000
```

---

Well, it could just be `connections_static`, changing the order should be enough for clarity.

---

Done